app/src/main/java/com/eveningoutpost/dexdrip/cgm/connectfollow/ConnectClient.java

```diff
@@ -0,0 +1,346 @@
+package com.eveningoutpost.dexdrip.cgm.connectfollow;
+
+import android.util.Log;
+
+import java.io.IOException;
+import java.util.Date;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import lombok.Getter;
+import okhttp3.*;
+
+import com.eveningoutpost.dexdrip.cgm.connectfollow.messages.ConnectData;
+import com.eveningoutpost.dexdrip.cgm.connectfollow.messages.ConnectDataResult;
+import com.google.gson.*;
+
+
+
+// TODO list
```

Comment:
Is this really a todo list or more likely a list of ideas? If these are todos and important, please implement them.

Reply:
Only future improvement options. I will remove them from the code.

Reply:
You can put them into the PR description and/or create a feature request with a todo list.